components/BottomAppBar/tests/snapshot/MDCBottomAppBarSnapshotTests.m

```diff
@@ -163,4 +163,40 @@ - (void)testFloatingButtonElevationSecondaryRTL {
   [self generateSnapshotAndVerifyForView:self.appBar];
 }
 
+- (void)testDynamicColorSupportOniOS13AndAbove {
+#if defined(__IPHONE_13_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_0)
+  if (@available(iOS 13.0, *)) {
+    // Given
+    UIColor *barTintDynamicColor =
+        [UIColor colorWithDynamicProvider:^(UITraitCollection *traitCollection) {
+          if (traitCollection.userInterfaceStyle == UIUserInterfaceStyleLight) {
+            return UIColor.blackColor;
+          } else {
+            return UIColor.purpleColor;
+          }
+        }];
+    UIColor *shadowDynamicColor =
+        [UIColor colorWithDynamicProvider:^(UITraitCollection *traitCollection) {
+          if (traitCollection.userInterfaceStyle == UIUserInterfaceStyleLight) {
+            return UIColor.blackColor;
+          } else {
+            return UIColor.blueColor;
+          }
+        }];
+    self.appBar.barTintColor = barTintDynamicColor;
+    self.appBar.shadowColor = shadowDynamicColor;
+
+    // When
+    self.appBar.overrideUserInterfaceStyle = UIUserInterfaceStyleDark;
+
+    // Then
+    CGSize aSize = [self.appBar sizeThatFits:CGSizeMake(360, INFINITY)];
+    self.appBar.bounds = CGRectMake(0, 0, aSize.width, aSize.height);
+    [self.appBar layoutIfNeeded];
+    UIView *snapshotView = [self.appBar mdc_addToBackgroundView];
```

Comment:
Tip: mdc_addToBackgroundViewInsets:UIEdgeInsets(50, 50, 50, 50);

Reply:
Just curious if the inset is needed here?

Reply:
I think it would allow us to see the shadow better.

Reply:
Addressed in f00da038a1f1c7b457c7764011f51d747c70b7b4.